group_vars/customer/patient0/development/main.yml

```diff
@@ -1,6 +1,7 @@
 # Variables specific to the patient0 customer in development env_type
 
 apps:
+  - name: mailcatcher
```

Comment:
Can you also add it to `env_types` where it is required (_i.e._ `group_vars/env_type/*`)?

Reply:
Done

Reply:
I only see CI and development, I thought we would use it everywhere but in production?

Reply:
@jmaupetit there was no "apps" parameter to add it to so I thought these environments are not yet used anyway.

Reply:
Ok fine. In this case, it means that every app is considered as deployable.